core-api/src/main/java/org/silverpeas/core/cache/model/AbstractSimpleCache.java

```diff
@@ -23,7 +23,10 @@
  */
 package org.silverpeas.core.cache.model;
 
+import java.util.Objects;
 import java.util.UUID;
+import java.util.function.Function;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Remove this unused import 'java.util.function.Function'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://integration.silverpeas.org/sonar/coding_rules#rule_key=squid%3AUselessImportCheck)